filament/src/Culler.cpp

```diff
@@ -22,11 +22,9 @@
 
 using namespace filament::math;
 
-// We use a vectorize width of 8 because, on ARMv8 it allows the compiler to write eight
-// 8-bits results in one go. Without this it has to do 4 separate byte writes, which
-// ends-up being slower.
-
-#define FILAMENT_CULLER_VECTORIZE_HINT 8
+// use 8 if Culler::result_type is 8-bits, on ARMv8 it allows the compiler to write eight
```

Comment:
I'm confused here.. isn't `Culler::result_type` now 16 bits?

Reply:
right it's 16 bits now, so we use a value of 4 -- if it was 8 bits we'd need to use 8 (that's what the comment is trying to say).